add new translation on r

arojascastro opened the pull request:
Attempting to add a new translation by @jenniferisasi 

Please approve after checking TRAVIS validation process.

arojascastro committed:
add new translation on r

arojascastro committed:
fix link

arojascastro committed:
fix link ?

arojascastro committed:
Update procesado-basico-de-textos-en-r.md

arojascastro committed:
fix two internal links

arojascastro committed:
add images

arojascastro committed:
fix images links

arojascastro commented:
Ok, all checks passed!

arojascastro commented:
@mdlincoln I'm sorry for bothering again, but I cannot see where is the problem. I get this warning:

```- ./_site/es/lecciones/index.html
  *  internal image /gallery//en/basic-text-processing-in-r.png does not exist (line 1018)
htmlproofer 3.9.0 | Error:  HTML-Proofer found 1 failure!
```

But I do not find `/basic-text-processing-in-r.png` in the file.

acrymble commented:
you have 2 slashes after /gallery

arojascastro commented:
But I do not find that `/gallery//en/basic-text-processing-in-r.png` in the text...

walshbr commented:
Looks like the error is coming up on a different page - the lessons index - based on the metadata for the lesson you added. I think it's this line from your metadata in the header of the file you added - line 20:

`original: /en/basic-text-processing-in-r`

Copying the format from the other Spanish lessons it looks like it should be -

`original: basic-text-processing-in-r`

Looks like it just takes the value of that key and tries to find the image in the galleries folder from that. But there's no 'en' folder in our gallery folder. So when you added the new lesson it is trying to find the image for the lesson but getting tripped up. Let me know if that doesn't make sense!

arojascastro committed:
change original metadata

arojascastro commented:
Thank you @walshbr - that was the error! :)

arojascastro commented:
@jenniferisasi I had to change some code containing a link:

From this


```{r}
base_url <- "https://programminghistorian.org/assets/basic-text-processing-in-r"
url <- sprintf("%s/sotu_text/236.txt", base_url)
texto <- paste(readLines(url), collapse = "\n")
```


To this 

```
+{r}
+base_url <- "/assets/basic-text-processing-in-r"
+url <- sprintf("%s/sotu_text/236.txt", base_url)
+texto <- paste(readLines(url), collapse = "\n")
+
```

Does it make sense to you? I guess you followed the original, but the new TRAVIS requirements forced me to make the path relative. Does still work in the context of the lesson? I hope.

mdlincoln commented:
Oh, no, the full link `https://programminghistorian.org/assets/basic-text-processing-in-r` should be ok when it's in the context of a code block. I thought I had adjusted the Travis checks to ignore those 😠  Give me a few minutes to check out why it was being triggered.

arojascastro commented:
Well maybe it was not alerting and I changed it because I thought that lied
the problem. I can undo and double check

-- 
​Dr. Antonio Rojas Castro
Researcher, Cologne Center for eHumanities
Communication coordinator, EADH
<http://www.antoniorojascastro.com>

El día 22 de junio de 2018 a las 16:47:18, Matthew Lincoln (
notifications@github.com) escrito:

> Oh, no, the full link
> https://programminghistorian.org/assets/basic-text-processing-in-r should
> be ok when it's in the context of a code block. I thought I had adjusted
> the Travis checks to ignore those 😠 Give me a few minutes to check out why
> it was being triggered.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/programminghistorian/jekyll/pull/922#issuecomment-399467864>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHAAT101OEqxMKivuPVbZ2zOcCXkqMdZks5t_QN2gaJpZM4UzdlI>
> .
>

mdlincoln committed:
restore full PH link for R code chunk

mdlincoln commented:
Yes, it looks like you changed two links in commit ad8e2a9560dc478b655909fa1d719c428932f0cf. The one in the R code was fine - it was only the one in the markdown text that was triggering the alert. I can try to update the code check to try and show the line number or context of the problem links so that it's easier to understand where the errors are.

mdlincoln committed:
no redirect-from needed in new translations

arojascastro commented:
Thank you all! The full link was restored by Matthew. If Travis is not arguing, can anyone approve the pull request please?

mdlincoln commented:
hrm, not yet - there is a redirect loop going on. Let me check the lesson metadata again

mdlincoln committed:
use backticks instead of html spans

mdlincoln committed:
translated images belong in original lesson folder

mdlincoln committed:
correct full link to datos-tabulares-en-r

mdlincoln committed:
correct full link to r-basics-with-tabular-data

mdlincoln commented:
I've pushed fixes for a few errors:

- We don't use `redirect_from` in translated lessons anymore
- Translated images still go into the image directory of the original lesson, so I moved the images into `/images/basic-text-processing-in-r`
- When using `{% incldue figure.html...` only put the name of the file, no other path needed.

mdlincoln commented on es/lecciones/procesado-basico-de-textos-en-r.md:
```diff
@@ -0,0 +1,628 @@
+# Notas
+
+[^1]: Nuestro corpus contiene 236 discursos del Estado de la Unión. Dependiendo de lo que se cuente, este número puede ser ligeramente más alto o más bajo.
+[^2]: Taryn Deward, [“Datos tabulares en R,” Programming Historian](/es/lecciones/datos-tabulares-en-r). Trad. de Jennifer Isasi de 10 de abril de 2018.
```
This translation does not appear to be published yet?

arojascastro replied:
It's currently under review: https://github.com/programminghistorian/ph-submissions/issues/164

I'm hoping to publish it in the following days. But yes we shouldn't linked it yet...

mdlincoln committed:
Correct spelling for Taryn Dewar

arojascastro committed:
add third reviewer

I forgto to

arojascastro committed:
fix name

arojascastro committed:
remove link

arojascastro commented:
I removed that internal link and now it looks all ok.

mdlincoln commented:
OK @arojascastro you might want to open an issue to remember to add the link back in when “Datos tabulares en R” in eventually published.

mdlincoln committed:
Merge branch 'gh-pages' into add-translation-r

arojascastro committed:
change name file

arojascastro commented:
I changed the name file and title from "procesado" to "procesamiento". I hope @jenniferisasi agrees.

arojascastro commented:
Sorry, I need someone to review and accept this merge :) thanks

mdlincoln committed:
Merge branch 'gh-pages' into add-translation-r

mdlincoln reviewed (approved):
I'm happy if @programminghistorian/spanish-team is happy 😄

mdlincoln commented:
Merge whenever you're ready @arojascastro
